.vscode/tasks.json

```diff
@@ -1,32 +1,95 @@
 {
-	// See https://go.microsoft.com/fwlink/?LinkId=733558
-	// for the documentation about the tasks.json format
-	"version": "2.0.0",
-	"tasks": [
-		{
-			"label": "Compile",
-			"type": "shell",
-			"command": "make -j`nproc` all",
-			"problemMatcher": "$gcc",
-			"group": {
-				"kind": "build",
-				"isDefault": true
-			},
-		},
-		{
-			"label": "Clean",
-			"type": "shell",
-			"command": "make -j`nproc` clean",
-			"problemMatcher": "$gcc",
-			"group": "build",
-		},
-		{
-			"label": "Generate compile_commands.json",
-			"type": "shell",
-			"command": "make clean-build clean-tools && bear -- make -j2 standard",
-			"problemMatcher": "$gcc",
-			"group": "build",
-			"dependsOn": ["Compile"],
-		}
-	]
-}
\ No newline at end of file
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Check For Game Directory",
+            "type": "shell",
+			"command": "echo -e '\\033[0;97mSource Code directory: ${workspaceFolder}\\033[0m'; if [ ! -d \"${config:game.directory}\" ]; then echo -e '\\033[0;91mGame directory missing in settings.json. Please update it.\\033[0m'; exit 1; else echo -e '\\033[0;97mGame directory: ' ${config:game.directory} '\\033[0m'; fi",
+			"runOptions": {
+                "runOn": "folderOpen"
+            },
+            "group": "build",
+            "presentation": {
+                "echo": false,
+                "focus": true,
+                "panel": "shared",
+                "showReuseMessage": false
+            }
+        },
+        {
+            "label": "Compile",
+            "type": "shell",
+            "command": "make -j`nproc` all && echo -e '\\033[0;32mCompilation successful!\\033[0m'",
```

Comment:
You dropped the problemMatcher.

Reply:
I've added it back but now I get error spam (for VSCode): "Unrecognized problem matcher. Is the extension that contributes this problem matcher installed?"
I don't understand what this is even for, or what extension I have to install.

Reply:
It's included in the recommended extensions. There should be a notification if it isn't installed.